Tech Journey/My-#100DayOfCode-Journey-So-Far.md

```diff
@@ -0,0 +1,56 @@
+## A bit about me, Who am I?
+Hey, my name is **Ameen**, I am 15 years old and currently in high school. I started learning to code when I was in second grade in school but never took it seriously until I was 11... That time I started learning **Unity** ( A Game Engine ) and **C#** but due to my PC's specs I couldn't continue on that path so instead of giving up  I started learning **Python**, learned **PyGame** and a few more libs and built something basic with each of em.
```

Comment:
second grade is enough, no need to add a school.  
you're comparing 2nd grade with age 11, use one thing. Either age or class you're in.
Avoid ... (triple dot)  
"That time I started learning Unity" can be written as "I started with Unity and C#" - the idea is to keep things as simple as possible, prefer active voice.

Reply:
Break the sentence of C# and start a new line. Break your sentences into a shorter one. Users won't be able to keep a lot of info at any given time. Make it easier for him to read your blog.

"**PyGame** and a few more libs and built" can be written as "**PyGame** with a few more libraries and built" - don't use too many ands